src/patternfly/components/ProgressStepper/progress-stepper.scss

```
@@ -0,0 +1,248 @@
+      @include pf-u-screen-reader;
+    }
+
+    .pf-c-progress-stepper__step-description {
+      display: none;
+    }
+
+    // For this compact variant only, move the main content for the current step to above the icons
+    // stylelint-disable
+    .pf-c-progress-stepper__step.pf-m-current .pf-c-progress-stepper__step-main {
+        grid-column: 1/-1;
+        grid-row: 1/2;
+      }
+    // stylelint-enable
+  }
+}
+
+// Step
+.pf-c-progress-stepper__step {
+  display: contents;
+
+  // Step Modifiers
+  &.pf-m-current {
+    --pf-c-progress-stepper__step-title--FontWeight: var(--pf-c-progress-stepper__step--m-current__step-title--FontWeight);
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-current__step-title--Color);
+  }
+
+  &.pf-m-pending {
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-pending__step-title--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--success-color--100);
+  }
+
+  &.pf-m-danger {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--danger-color--100);
+    --pf-c-progress-stepper__step-title--Color: var(--pf-c-progress-stepper__step--m-danger__step-description--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--warning-color--100);
+  }
+
+  &.pf-m-info {
+    --pf-c-progress-stepper__step-icon--Color: var(--pf-global--info-color--100);
+  }
+}
+
+// The step icon wrapper provides a container on which to use the before for the line connecting the steps
+.pf-c-progress-stepper__step-connector {
+  position: relative;
+  display: flex;
+  justify-content: var(--pf-c-progress-stepper__step-connector--JustifyContent);
+  width: 100%;
+  transform: translateX(0);
+
+  .pf-c-progress-stepper__step:not(:last-of-type) > &::before {
+    position: absolute;
+    top: var(--pf-c-progress-stepper__step-connector--before--Top); // half the height
+    left: var(--pf-c-progress-stepper__step-connector--before--Left);
+    z-index: -1;
+    width: var(--pf-c-progress-stepper__step-connector--before--Width);
+    height: var(--pf-c-progress-stepper__step-connector--before--Height);
+    content: "";
+    border-right: var(--pf-c-progress-stepper__step-connector--before--BorderRight);
+    border-bottom: var(--pf-c-progress-stepper__step-connector--before--BorderBottom);
+  }
+}
+
+// Step icon
+.pf-c-progress-stepper__step-icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: var(--pf-c-progress-stepper__step-icon--Width);
+  height: var(--pf-c-progress-stepper__step-icon--Height);
+  font-size: var(--pf-c-progress-stepper__step-icon--FontSize);
+  color: var(--pf-c-progress-stepper__step-icon--Color);
+  background-color: var(--pf-c-progress-stepper__step-icon--BackgroundColor);
+  border: 2px solid var(--pf-c-progress-stepper__step-icon--BorderColor);
```

var(--pf-global--BorderWidth--md) is 2px

```suggestion
  border: 2px solid var(--pf-c-progress-stepper__step-icon--BorderColor);
```